drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/mci.rb

```diff
@@ -106,7 +106,7 @@ def create_mci_id(client)
     def update_client(client)
       raise(Error, 'Client needs to be saved first') unless client.persisted?
 
-      external_id = get_external_id(client)
+      external_id = client.mci_id
```

Comment:
It looks like `get_external_id` returned an ExternalId, but `mci_id` returns a string

Reply:
Thats a great spot, thank you!